src/services/ContributorAuditService/utilities.ts

```diff
@@ -15,4 +15,28 @@ const ParamUtilities = {
   },
 };
 
-export { ParamUtilities };
+const DataMappingUtilities = {
+  mergeContributors(contributorsArray: any[][]): any[] {
```

Comment:
this should be typed - you can do a shared interface to make sure the username / id etc is available